### PR TITLE
refactor emsdk installation

### PR DESCRIPTION
**Pull request by Ludea**
using docker locally to build package

### CODE REVIEW

File: packages/emsdk/build.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+TERMUX_PKG_DEPENDS="llvm"
+TERMUX_PKG_BUILD_IN_SOURCE=true
+
+termux_step_post_get_source() {
+	./emsdk install latest 
+	./emsdk activate latest 	
+}
+
+termux_step_make_instal() {
+	rm $TERMUX_PREFIX/lib/emsdk -rf
```
Comment (Ludea) [inline packages/emsdk/build.sh:16]: not sure if is the right path

Comment (Zack-sys) [reply packages/emsdk/build.sh:16]: Please comment the command I can't copy the the command

**Comment by Grimler91** [comment]
emsdk is not arch-independent is it? 

Looks like there is nothing selecting a specific architecture in this PR, so I guess `./emsdk install latest` will install emsdk for x86_64, which will not work on aarch64, arm, i686 (and not x86_64 either since it will not be linked against android's bionic).

**Comment by Ludea** [comment]
I found https://github.com/emscripten-core/emsdk/blob/master/emsdk.py#L117, so suppose emsdk detect and download its dependencies on right arch

**Comment by Grimler91** [comment]
> I found https://github.com/emscripten-core/emsdk/blob/master/emsdk.py#L117, so suppose emsdk detect and download its dependencies on right arch

But when cross-compiling the machine type will always be x86_64, so it won't download for the correct architecture.

File: packages/emsdk/build.sh (hunk 1 above)
Comment (xeffyr) [reply packages/emsdk/build.sh:16]: > termux_step_make_instal()

This function is not being executed due to typo.

Comment (Ludea) [reply packages/emsdk/build.sh:16]: Good catch

**Comment by Ludea** [comment]
I'm looking for download the right dependencies. 
I can not specify the arch into the install script

**Comment by Ludea** [comment]
I will refactor the installation. It's not the right way for now

**Pull request closed without merging**